### DISCUSSION TITLE
Vite+React GitHub Page Not Showing

### DISCUSSION THREAD

**Discussion by dannypeelen** [community/community#148001] [Programming Help] [answered]
### Body

I am currently trying to deploy my personal webpage that I built through Vite+React using GitHub. When I run it locally, everything works as is, but when I push it to my repository and it deploys, the page is blank.

Here's my repository: [https://github.com/dannypeelen/dannypeelen.github.io](url)

I'm not sure what is wrong, any help would be appreciated.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by lemorage** [comment]
The [page](https://dannypeelen.github.io/) seems normal now...

**Reply by dannypeelen** [reply]
Yes, sorry, I managed to fix it. Turned out I just need to deploy from my gh-pages branch.

**Comment by dannypeelen** [comment, marked as answer]
I went into the settings of my repository and chose to deploy from the gh-pages branch.
